cs-couchbase/src/main/java/io/cloudslang/content/couchbase/actions/cluster/GetAutoFailOverSettings.java

```diff
@@ -0,0 +1,169 @@
+     * https://developer.couchbase.com/documentation/server/3.x/admin/REST/rest-cluster-autofailover-settings.html
+     *
+     * @param endpoint             Endpoint to which request will be sent. A valid endpoint will be formatted as it shows in
+     *                             bellow example.
+     *                             Example: "http://somewhere.couchbase.com:8091"
+     * @param username             Username used in basic authentication.
+     * @param password             Password associated with "username" input to be used in basic authentication.
+     * @param proxyHost            Optional - proxy server used to connect to Couchbase API. If empty no proxy will be used.
+     * @param proxyPort            Optional - proxy server port. You must either specify values for both proxyHost and
+     *                             proxyPort inputs or leave them both empty.
+     * @param proxyUsername        Optional - proxy server user name.
+     * @param proxyPassword        Optional - proxy server password associated with the proxyUsername input value.
+     * @param trustAllRoots        Optional - specifies whether to enable weak security over SSL/TSL. A certificate is
+     *                             trusted even if no trusted certification authority issued it.
+     *                             Valid values: "true", "false"
+     *                             Default value: "true"
+     * @param x509HostnameVerifier Optional - specifies the way the server hostname must match a domain name in the subject's
+     *                             Common Name (CN) or subjectAltName field of the X.509 certificate. Set this to "allow_all"
+     *                             to skip any checking. For the value "browser_compatible" the hostname verifier works
+     *                             the same way as Curl and Firefox. The hostname must match either the first CN, or any
+     *                             of the subject-alts. A wildcard can occur in the CN, and in any of the subject-alts.
+     *                             The only difference between "browser_compatible" and "strict" is that a wildcard (such
+     *                             as "*.foo.com") with "browser_compatible" matches all subdomains, including "a.b.foo.com".
+     *                             Valid values: "strict", "browser_compatible", "allow_all"
+     *                             Default value: "allow_all"
+     * @param trustKeystore        Optional - pathname of the Java TrustStore file. This contains certificates from other
+     *                             parties that you expect to communicate with, or from Certificate Authorities that you
+     *                             trust to identify other parties. If the protocol (specified by the "url") is not "https"
+     *                             or if trustAllRoots is "true" this input is ignored.
+     *                             Default value: ../java/lib/security/cacerts
```

Comment:
Not sure about this default value: isn't it taken from system as all workers will have -Djavax.net.ssl.trustStore property set during initilization.

Reply:
Ran it against a secured cluster (https, with a certificate to authenticate it) added that certificate here: <java_jdk_instalation_folder>\jre\lib\security -> works as expected.

Reply:
Of course, those default values means least secured call...

Reply:
Thanks for the information @mihaitusa, I marked the comment as resolved.